MarvellOS.py

```diff
@@ -6,7 +6,16 @@
 from tkinter import *
 import base64
 
-WINDOW_DIM = "325x400"
+
+#Tkinter Related
+
+import tkinter as tk
+import tkinter.filedialog as tkfd
+from tkinter import *
+from tkinter import ttk 
```

Comment:
Try to import them all at once using something like this: `from math import (atan as t, degree as z)`.